### PR TITLE
Fix/updates

### PR DESCRIPTION
**Pull request by bgantick**

### CODE REVIEW

File: src/sass/objects/_grid.scss [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    }
+    .o-column--order-#{$i} {
+      order: $i;
+    }
+  }
+}
+
+@include generate-columns(12);
+
+.o-column--third {
```
Comment (austinstarin) [inline src/sass/objects/_grid.scss]: I think we can remove this.

File: src/sass/objects/_grid.scss [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    width: (100% / 3);
+  }
+  &.o-column--break-sm {
+    @include media('>small') { /* stylelint-disable-line */
+      width: (100% / 3);
+    }
+  }
+}
+
+.o-column--quarter {
```
Comment (austinstarin) [inline src/sass/objects/_grid.scss]: I think we can remove this.

File: src/sass/objects/_grid.scss [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+}
+
+@mixin generate-columns($number) {
+  @for $i from 1 through $number {
+    .o-column-#{$i} {
+      width: 100%;
+      @include media('>medium') {
+        width: (100% * $i / $number);
+      }
+      &.o-column--break-sm {
```
Comment (austinstarin) [inline src/sass/objects/_grid.scss]: I think we need to consider these conditions further...

Maybe we auto-generate these break conditions using the breakpoints map?

File: src/sass/objects/_grid.scss [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+.o-grid--justify {
+  justify-content: space-between;
+}
+
+.o-grid--stretch {
+  align-items: stretch;
+}
+
+.o-grid--gutter {
```
Comment (austinstarin) [inline src/sass/objects/_grid.scss]: We should introduce a mixin for gutters that takes a map.

**Comment by austinstarin** [comment]
@bgantick Let's merge this into `develop`. Also, if it makes sense to break these changes into smaller, more focused merges let's do that.

File: src/sass/objects/_grid.scss (hunk 3 above)
Comment (bgantick) [reply src/sass/objects/_grid.scss]: Maybe open an issue for this?

File: src/sass/objects/_grid.scss (hunk 4 above)
Comment (bgantick) [reply src/sass/objects/_grid.scss]: Should open an issue for this too

**Pull request merged**